timemachine/cpp/src/barostat.hpp

```diff
@@ -30,11 +31,21 @@ template <typename RealType> class MonteCarloBarostat {
 
     int get_interval();
 
+    double get_volume_scale_factor();
+
+    void set_volume_scale_factor(const double scaling);
```

Comment:
nit:
```suggestion
    void set_volume_scale_factor(const double scale_factor);
```

Reply:
Fixed in [b102405](https://github.com/proteneer/timemachine/pull/1152/commits/b10240596853c2ab57047e8b153a6536fe373412)